index.js

```diff
@@ -145,6 +145,14 @@ alexa.request = function(json) {
       return null;
     }
   };
+  // session is not included for AudioPlayer or PlaybackController requests
```

Comment:
Maybe add the https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/alexa-skills-kit-interface-reference#request-body-parameters link here too?